canhttp/src/lib.rs

```diff
@@ -9,6 +9,8 @@ pub use client::{Client, IcError};
 pub use cycles::{
     CyclesAccounting, CyclesAccountingError, CyclesChargingPolicy, CyclesCostEstimator,
 };
+pub use observability::ObservabilityLayer;
```

Comment:
nit: this fixes a few doc warnings:
```suggestion
pub use observability::{Observability, ObservabilityLayer, RequestObserver, ResponseObserver};
```